app/views/statuses/_form.html.erb

```diff
@@ -31,12 +31,7 @@ See COPYRIGHT and LICENSE files for more details.
 
 <section class="form--section">
   <div class="form--field -required"><%= f.text_field "name", required: true, container_class: "-middle" %></div>
-  <div class="form--field">
-    <%= f.select :default_done_ratio,
-                 (0..100).step(10).map { |r| ["#{r} %", r] },
-                 container_class: "-xslim",
-                 label: WorkPackage.human_attribute_name(:done_ratio) %>
-  </div>
+  <div class="form--field"><%= f.number_field "default_done_ratio", min: 0, max: 100, container_class: "-tiny" %></div>
```

Comment:
The input field is too small in Firefox MacOS, it doesn't fit 2 digit numbers, due to the stepper arrows.

<img width="774" alt="image" src="https://github.com/user-attachments/assets/593a00e5-a3fe-47d9-884b-4c2eee965f2d">

Reply:
good catch, I reduced it to a minimum, but did not check with all browsers. I'll fix that.

Reply:
@dombesz It should be fixed now. Can you test it again please?